packages/manager/modules/ufrontend/url-resolver.js

```diff
@@ -0,0 +1,20 @@
+export const resolveUrls = (routes) => {
+  return fetch('/engine/2api/url-resolver', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(routes),
```

Comment:
**question:** Is `credentials: 'same-origin',` not needed in this case?

Relates to a previous Pull Request if I'm not wrong.
see: https://github.com/ovh/manager/pull/2719

Now it lives in `@ovh-ux/manager-config` module.
see: https://github.com/ovh/manager/blob/c9df248543a739176a8253b17473010b4c636d1e/packages/manager/modules/config/src/index.js#L35

Reply:
Nope because we don't need to be logged to call this 2API.